<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>491</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Transmittance parameters</string>
    </property>
    <widget class="QLineEdit" name="a2Edit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>41</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a1Edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>41</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a0Edit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>41</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>30</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>a2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>30</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>a1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>30</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>a0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>41</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>30</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>T</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UpdateButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>104</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Input parameters</string>
    </property>
    <widget class="QLineEdit" name="freqEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>0.001</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ampEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phaseEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>PI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>freq</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>amplitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>phase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateInputButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>104</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="squareRadio">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>S&amp;quare</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="triangleRadio">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Trian&amp;gle</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="sineRadio">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>S&amp;ine</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="controlBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startStopButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rescaleButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Rescale Axes</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="statusBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>a2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>a1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>a0:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>freq:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>phase:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>amplitude:</string>
     </property>
    </widget>
    <widget class="QLabel" name="a2Label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="a1Label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="a0Label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="phaseLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="freqLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ampLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Stable:</string>
     </property>
    </widget>
    <widget class="QLabel" name="stableLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Input:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>T:</string>
     </property>
    </widget>
    <widget class="QLabel" name="TLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="inputLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1325</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
